bedrock/settings/base.py

```diff
@@ -1337,6 +1343,11 @@ def JINJA_CONFIG():
     'firefox_os_mwc_2015_preview': {
         'source_filenames': (
             'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/matchMedia.js',
+            'js/libs/matchMedia.addListener.js',
```

Comment:
I think `'js/libs/matchMedia.addListener.js'` also needs to be in a conditional comment, to avoid errors in IE8 and below.